### ISSUE TITLE
Store repository selections in local storage

### ISSUE THREAD

**Issue by kthaler** [iFixit/pulldasher#409]
## Description
There are a few repositories that have a lot of activity, but don't need QA/CR, so they end up cluttering pulldasher. While we can currently select the repositories we want to monitor, these selections are not saved, requiring us to reselect them each session.

## Do this
Save the repo selection to local storage so we only need to select the repos we care about once.

**Comment by danielbeardsley** [comment]
> these selections are not saved, requiring us to reselect them each session.

I don't think this is the case (at least anymore). Repo selections (like all other selections) are saved in the url so every view is bookmarkable:

https://pulldasher.cominor.com/?repo=expo%2Cfixbot

I think storing the selections in local storage is a bit frustrating. Then there's only one global set of current settings and temporarily changing your view alters those global settings.
